.vsts-pr.yml

```diff
@@ -58,6 +58,18 @@ stages:
         vmImage: macOS-latest
         os: macOS
       helixTargetQueue: osx.13.amd64.open
+  ### ARM64 ###
+  - template: /eng/pipelines/templates/jobs/sdk-job-matrix.yml
+    parameters:
+      pool:
+        name: Azure Pipelines
+        vmImage: macOS-latest
+        os: macOS
+      helixTargetQueue: osx.13.arm64.open
+      macOSJobParameterSets:
+      - categoryName: TestBuild
+        buildArchitecture: arm64
+        runtimeIdentifier: osx-arm64
```

Comment:
Defining `macOSJobParameterSets` here overrides the default in `sdk-job-matrix.yml`. Since we only specify a single entry, it will only create a single job. This is also how the design of the Official pipeline works; it defines a bunch of specific job sets to do the official builds.